### ISSUE TITLE
Issue when deploying

### ISSUE THREAD

**Issue by garybushey** [hunters-forge/Blacksmith#5]
I get this error when trying to deploy to the Sentinel2Go to Azure:

Deployment template validation failed: 'The template parameters 'workspaceId, securityCollectionTier' in the parameters file are not valid; they are not present in the original template and can therefore not be provided at deployment time. The only supported parameters for this template are 'workspaceName, location'. Please see https://aka.ms/arm-deploy/#parameter-file for usage details.'. (Code: InvalidTemplate)

**Comment by Cyb3rWard0g** [comment]
Hey @garybushey ! Would you mind sharing how you are deploying Sentinel2Go?

**Comment by Cyb3rWard0g** [comment]
I think I know what parameters you were using while Deploying Sentinel2Go. You might have tried to use the `EnableWinDatatSources` flag. That is part of Sentinel2Go Part 2 blog post :) I did not talk about that one in the first blog post because that ones was not ready yet. I fixed the `EnableWinDataSources` parameter anyways with the commit below. Let me know if that was the parameter that you were trying to use and got the error message from. thank you in advance @garybushey !
